src/commands/api-mesh/__tests__/create.test.js

```diff
@@ -82,6 +83,12 @@ describe('create command tests', () => {
 		});
 		subscribeCredentialToMeshService.mockResolvedValue(['dummy_service']);
 
+		let fetchedMeshConfig = sampleCreateMeshConfig;
+		fetchedMeshConfig.meshConfig.meshId = 'dummy_id';
+		fetchedMeshConfig.meshConfig.meshURL = '';
```

Comment:
Similarly, I think neither `meshId` nor `meshUrl` are contained in the `meshConfig`. If that's not the case, let's make sure to update the mocked value from `getMesh`. I see it in multiple locations in this file.

Reply:
yeah I corrected it now @amolina-adobe

Reply:
Looks like you only corrected it for `describe.test.js`. Can you please make the changes here too?

Reply:
yes. updated tests now